packages/serverpod/lib/src/database/database_query.dart

```diff
@@ -605,6 +664,7 @@ class _SubQuery {
 class _SubQueries {
   static const String orderByPrefix = 'order_by';
   static const String whereCountPrefix = 'where_count';
+  static const String whereNonePrefix = 'where_any';
```

Comment:
Chore: Wrong word used for prefix.
```suggestion
  static const String whereNonePrefix = 'where_none';
```